crates/bevy_core/src/time/timer.rs

```diff
@@ -464,4 +465,20 @@ mod tests {
         t.tick(Duration::from_secs_f32(0.5));
         assert_eq!(t.times_finished(), 0);
     }
+
+    #[test]
+    fn times_finished_precise() {
+        let mut t = Timer::from_seconds(0.01, true);
+        let duration = Duration::from_secs_f64(1.0 / 3.0);
+
+        t.tick(duration);
+        assert_eq!(t.times_finished(), 33);
+        t.tick(duration);
+        assert_eq!(t.times_finished(), 33);
+        t.tick(duration);
+        assert_eq!(t.times_finished(), 33);
```

Comment:
shouldn't the extra tick be on this one?

Reply:
So my logic was roughly this: 

0 + 33.3333 = 33.3333 (Rounds down to 33 times finished, leaving the remainder as .3333)
.3333 + 33.3333 = 33.6666 (Rounds down to 33 again, remainder fo .6666)
.6666 + 33.3333 = 33.9999 (Rounds down last time to 33 with remainder of .9999)
.9999 + 33.3333 = 34.3332 (Rounds down to 34 this time)

Reply:
It will add the extra tick after the fourth iteration, then it will correctly add it every three iterations...  I would have liked it to be every three from the start, but that's probably not possible when using those kind of numbers 🤷

Reply:
Gotta love floating point numbers 🙃